ONSdigital/eq-author-app
# 🛠 EAR 556 - Fix onblur post 🐲💧 changes

### What is the context of this PR?

Modified inputs were not saving as the drag and drop library used
prevents propagation of events when the draggable items are clicked so
the `onBlur` handler never fired.

Apply workaround per GH issues page for `react-beautiful-dnd` to focus
draggable links on click so blur events are manually triggered.

### How to review

- Open / create a questionnaire
- Type a new question title or answer label but keep focus within the input element
- Switch to a new page
- Question title / answer label should update appropriately with the new value

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
